### PR TITLE
tests: Replacing distutils with site

### PR DESCRIPTION
**Pull request by thopiekar**
This gives us optionally access to the user location.

PS: As found while working on #1391

### CODE REVIEW

File: pywin32_testall.py [hunk 1]
```diff
@@ -62,27 +62,23 @@ def find_and_run(possible_locations, script, cmdline_rest=""):
                            % (script, possible_locations))
 
 if __name__=='__main__':
+    code_directories = [this_dir, ] + site_packages
```
[the remark this answers is not in the document]
Comment (thopiekar) [reply pywin32_testall.py]: Don't know why, but it feels like write is 70:30 better for me. Have no clear opinion about this and my PEP8 checker (hopefully correctly set up) is also no complaining. Therefore, if you wish I'll remove it of course ASAP :wink:

[the remark this answers is not in the document]
Comment (mhammond) [reply pywin32_testall.py]: Thanks - I've never seen a python tutorial or anything else recommend this style, so it just makes the casual reader do a double-take and wonder for a second what they are missing...

[the remark this answers is not in the document]
Comment (thopiekar) [reply pywin32_testall.py]: Ok, will change it then 😀

**Comment by thopiekar** [comment]
Thank you for the review!

**Pull request merged**